modules/drivers/snowflake/test/metabase/driver/snowflake_test.clj

```diff
@@ -109,7 +109,7 @@
                          :database-type     "VARCHAR"
                          :base-type         :type/Text
                          :database-position 1
-                         :database-required false}}}
+                         :database-required true}}}
```

Comment:
I have no idea why this is changed. if someone could shed some lights that'd be great.